Remotion/Web/Development.WebTesting/Utilities/AssertionExceptionUtility.cs

```diff
@@ -28,62 +29,70 @@ public static class AssertionExceptionUtility
   {
     [NotNull]
     [MustUseReturnValue]
-    public static WebTestException CreateControlDisabledException ([CallerMemberName] string operationName = "")
+    public static WebTestException CreateControlDisabledException (IDriver driver, [CallerMemberName] string operationName = "")

```

Comment:
When all is done, I'm thinking we may be able to do a not-null on trunk and have an overload without the driver for the backport.